source/scripting_v3/GTA/Entities/Vehicles/HandlingData.cs

```diff
@@ -1307,6 +1330,29 @@ public float WeaponDamageMultiplier
 			}
 		}
 
+		public float WeaponDamageScaledToVehicleHealthMultiplier
```

Comment:
So this one is for `fWeaponDamageScaledToVehHealthMult`, which is used only for the Rhino and Khanjali in vanilla handling.meta files.